src/app/presentation/feedback-slide/feedback-slide.component.ts

```diff
@@ -10,19 +10,24 @@ import {SlideComponent} from '../slide/slide.component';
 
 export class FeedbackSlideComponent implements OnInit {
 
+  // firstVisit: boolean = true;
+
   constructor(private presentation: PresentationComponent, private slide: SlideComponent) {
+    //  this.firstVisit = true;
   }
 
   ngOnInit() {
+  localStorage.setItem('visitedBefore', 'yes');
   }
 
   closeSlide() {
-    this.presentation.nextSlide();
+    // this.presentation.nextSlide();
```

Comment:
Sorry, probably not; this PR isn't commit-ready yet; still haven't figured out how to get the one-time display to work and still be able to navigate to other slides.